client/my-sites/stats/hooks/use-notice-visibility-query.ts

```diff
@@ -32,6 +33,10 @@ const CONFLICT_NOTICE_ID_GROUPS: Record< string, Array< NoticeIdType > > = {
 		'client_free_plan_purchase_success',
 		'do_you_love_jetpack_stats',
 		'commercial_site_upgrade',
+		// Temporarily set the order higher than the TierUpgradeNotice
+		// that would probably not show up by checking inside itself.
+		'gdpr_cookie_consent',
```

Comment:
I was imagining that the notice might need to be of top priority, because if Stats is not working for the users, they would never consider upgrading to paid? What do you think?

Reply:
Indeed, I agree with it.

Reply:
Agree as well! Please, be sure this overrides any other notice 🙏

Reply:
Addressed in [15879b2](https://github.com/Automattic/wp-calypso/pull/91762/commits/15879b27721ba1a9e513eb19a6c1dcbe7fb49280).